src/test/kotlin/no/nav/syfo/service/PapirSmMapperServiceSpek.kt

```diff
@@ -23,8 +23,7 @@ object PapirSmMapperServiceSpek : Spek({
     val dokumentInfoId = "123"
     val fnrPasient = "12345678910"
     val aktorId = "aktorId"
-    val datoOpprettet = LocalDateTime.now()
-    val loggingMetadata = LoggingMeta(sykmeldingId, journalpostId, "hendelsesId")
+    val datoOpprettet = OffsetDateTime.now()
```

Comment:
Lurer på om du må ha med zone offset, altså slik? 
`OffsetDateTime.now(ZoneOffset.UTC)`

Eller, må og må, det er kanskje ikke så nøye hvilken sone som brukes i testen uansett 😄

Reply:
Nei, jeg tester ikke den funksjonaliteten så det har ingenting å si, men jeg kan godt gjøre det uansett for å være konsekvent :)